projects/storefrontstyles/scss/themes/calydon/components/content/banner/_banner.scss

```diff
@@ -0,0 +1,6 @@
+%cx-banner-calydon {
+  &:focus-within {
+    z-index: 1;
+    box-shadow: 0px 0px 4px 4px var(--cx-color-visual-focus);
```

Comment:
I don't like the visual side of this fix - it is completely different than other focus states. but it fixes the problem for now, so we can focus on fixing that later